fhireval/data/EIII-Examples.json

```diff
@@ -0,0 +1,12894 @@
+{
```

Comment:
Given this is a test data set, it seems quite big. Since POSTing these records occurs via REST, it may take too long in loading?

Reply:
These are loaded individually. It does take a long time for them to fully appear, but there is code that blocks until all insertions and deletions have completed.